packages/jest-cli/src/cli/args.ts

```diff
@@ -377,8 +377,8 @@ export const options = {
   moduleNameMapper: {
     description:
       'A JSON string with a map from regular expressions to ' +
-      'module names that allow to stub out resources, like images or ' +
-      'styles with a single module',
+      'module names or an array of module names that allow to stub ' +
```

Comment:
```suggestion
      'module names or to arrays of module names that allow to stub ' +
```
(as in 'map from regular expressions to [...] arrays of module names')

Reply:
(also in other file)